firebase-firestore/src/test/resources/json/listen_spec_test.json

```diff
@@ -99,6 +99,7 @@
         "watchEntity": {
           "docs": [
             {
+              "createTime": 0,
```

Comment:
I know this is annoying, but can you remove all of the `createTime` lines from this file, since they are not part of the bloom filter tests?

Reply:
I'm not sure what happened, but there are still a bunch of `"createTime": 0` lines in this file. Could you remove them too?